circle.yml

```diff
@@ -6,9 +6,11 @@ dependencies:
     - ~/.android
     - ~/android
     - "~/assets/"
+    - ~/tomtom_assets
   pre:
     - "[ -d ~/assets ] || mkdir ~/assets"
     - "[ -d ~/android ] || mkdir ~/android"
+    - "[ -d ~/tomtom_assets ] || mkdir ~/tomtom_assets"
```

Comment:
We should probably take care of that in the tomtom specific script.

Eventually we should simplify the circle.yml file so that it only triggers sub jobs, but it should not really care about the jobs dependencies.